internal/app/machined/pkg/runtime/v1alpha1/platform/equinixmetal/testdata/expected.yaml

```diff
@@ -72,6 +72,7 @@ routes:
       flags: ""
       protocol: static
       layer: platform
+      mtu: 0
```

Comment:
we could add `"yaml:"mtu,omitempty"` to avoid this, and as it's rarely being used, that might make sense